### PR TITLE
EDU-908 Create memory display component

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-908

Screen Shot 2022-11-03 at 10 34 22

### CODE REVIEW

File: src/plugins/memory/memory.less [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  background-color: @edu-background-color-light;
+
+  &.MemoryDisplay--3x3 {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+  }
+
+  &.MemoryDisplay--4x4 {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
```
Comment (ahelmberger) [inline src/plugins/memory/memory.less:17]: Just nitpicking here, but `repeat(4, 1fr)` would read nicer :-)

**Review by ahelmberger — APPROVED** [review]

**Comment by ahelmberger** [comment]
Looks and feels super-nice :heart_eyes:

File: src/plugins/memory/memory.less (hunk 1 above)
Comment (adrianaluchian) [reply src/plugins/memory/memory.less]: True, much better than repeating 4 times 🤦‍♀️ Thanks, improved.

**Comment by adrianaluchian** [comment]
> Looks and feels super-nice 😍

Thihi ☺️

**Pull request merged**